### PR TITLE
Google play store link added, about screen changed

### PR DESCRIPTION
**Pull request by mananwason**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <string name="events_download_empty">No new events found</string>
 
     <plurals name="events_download_completed">
-        <item quantity="one">Completed downloading %1$d event</item>
+        <item quantity="one">Completed dcownloading %1$d event</item>
```
Comment (tradzik) [inline app/src/main/res/values/strings.xml]: `dcownloading` ?

**Comment by mananwason** [comment]
@ty221 sorry I'll correct this.

**Comment by tradzik** [comment]
@mananwason Thank you. Can you rebase your work to one commit please, if that is not any problem?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -116,11 +116,15 @@
     <string name="about">About</string>
     <string name="about_text">Programmed by <a href="http://www.digitalia.be/">Christophe Beyls</a>.
```
Comment (tradzik) [inline app/src/main/res/values/strings.xml:117]: How about `Programmed by <a href="http://www.digitalia.be/">Christophe Beyls</a> and FOSSASIA Community (<a href="https://github.com/fossasia/fossasia-companion-android/blob/master/README.md">check contributors</a>)` ?

**Comment by mananwason** [comment]
Yeah I guess this would be cool !!

**Comment by mananwason** [comment]
@ty221 where should the base be for the rebase?

**Comment by tradzik** [comment]
Please rebase all your commits to one comparing them to current `master` branch status.

**Comment by mananwason** [comment]
ok thanks :D

**Comment by mananwason** [comment]
@ty221 I did rebase it, but could you please check. Thanks :D

**Comment by tradzik** [comment]
@mananwason Hi, it seems there are still 5 commits here. Please follow `cherry-picking` and `rebasing` instrunctions as here: http://stackoverflow.com/questions/14635672/git-rebase-a-single-commit best luck :D

**Comment by mananwason** [comment]
thanks @ty221 :D

**Comment by tradzik** [comment]
@mananwason Just curiosity - why did you close this PR?

**Comment by mananwason** [comment]
@ty221 I was trying rebasing, I was not sure what to do. It says rebasing done, but doesn't do anything here.

**Pull request closed without merging**